haml/_footer.html.haml

```diff
@@ -29,5 +29,5 @@
         .cads-footer__logo
           %a.cads-logo{ href: homepage_url, title: t("cads.shared.logo_title") }
         .cads-footer__meta
-          %p.cads-footer__meta-text= t("cads.shared.copyright")
-          %p.cads-footer__meta-text= t("cads.shared.legal_summary")
+          %p.cads-footer__meta-text{ data: { test_id: "copyright" }}= t("cads.shared.copyright")
+          %p.cads-footer__meta-text{ data: { test_id: "legal summary" }}= t("cads.shared.legal_summary")
```

Comment:
Could we use `data-testid` rather than `data-test-id`. That way we're consistent with the conventions used in https://testing-library.com/docs/dom-testing-library/api-queries#bytestid for our DOM tests. And mind making the values hyphenated, e.g "legal-summary", rather than including spaces?

Reply:
Will update the code. And yes to your question, it does make it semi-circular. However this in practice is good with very simple static stuff like this, because eventually down the line we should be getting to me reviewing all tickets going into production (if not in code, just testing them). That way I could see if the logic was poor.

As for whether this would trap a poor label, the answer is no it wouldn't. But having 2 sets of yml files wouldn't either, because we would just update both of them simultaneously. That kind of logic is a bit untestable.

Reply:
Interesting. I don't entirely follow the notion of having a manual gate keeper vs relying on static tests but I think I get what you mean. Nothing blocking though as we seem to be using this I18n pattern across the board anyway.